dist/steps/build.sh

```diff
@@ -168,7 +148,7 @@ fi
 
 echo ""
 echo "###########################"
-echo "#       Build output      #"
+echo "#     Build directory     #"
```

Comment:
I would leave this as output, as the way of showing the output might be done in a less crude way than `ls` later on. Like with a screenshot.